java/src/jmri/Section.java

```diff
@@ -168,7 +171,7 @@ public int getState() {
 
     /**
      * Set the state of the Section.
-     *
+     * FREE, FORWARD, or REVERSE. UNKNOWN state not accepted here.
```

Comment:
To make this Javadoc easier to read, I suggest that you do:
```
* FREE, FORWARD, or REVERSE.
* <br>
* UNKNOWN state not accepted here.
```
When I first read this comment, I read it as none of FREE, FORWARD, REVERSE or UNKNOWN was valid.

Reply:
Thanks, definitely clearer.